lib/mix/test/mix/dep_test.exs

```diff
@@ -248,15 +248,14 @@ defmodule Mix.DepTest do
     @behaviour Mix.RemoteConverger
 
     def remote?(_app), do: true
+    def deps(_dep, _lock), do: []
+    def post_converge, do: :ok
```

Comment:
No it doesn't matter, it just looks a bit off in contrast with the rest of the codebase, but yeah you make good points as well :)